app/code/Magento/InventoryInStorePickupFrontend/view/frontend/web/js/model/pickup-locations-registry.js

```diff
@@ -0,0 +1,32 @@
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+
+define([], function() {
```

Comment:
Generally, this file is redundant. From my point of view, cache on the frontend isn't a good idea. Also, it's not real cache, it looks like temporary storage for some data, and we should use DataSource for these cases if we really need it.